easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.2-foss-2016b.eb

```diff
@@ -0,0 +1,26 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
```

Comment:
@SethosII I agree the situation is a bit of a mess at present, and we should agree on a format then document it. 

In the meantime, I just think it's better not to alter attributions where they already exist.